### DISCUSSION TITLE
Is it possible to trigger and event when after highlighting multiple cells via ExcelCopyBuffer?

### DISCUSSION THREAD

**Discussion by ptatoChi** [ghiscoding/Angular-Slickgrid#1165] [Ideas]
Its cool to select multiple cells via ExcelCopyBuffer. I have a usecase that doesnt involve copy/paste.  We just want to have an event triggered containing the cell data after selecting x number of cells

**Comment by ghiscoding** [comment]
You can get the cell ranges with `onCopyCells`, see all events in this [interface](https://github.com/ghiscoding/slickgrid-universal/blob/3ebde547b74c3f4944c327de4d61cc04fbfc4e71/packages/common/src/interfaces/excelCopyBufferOption.interface.ts#L56-L66)
